acsoo/addons.py

```diff
@@ -74,3 +76,47 @@ def addons_list_depends(ctx, exclude):
 
 
 addons.add_command(addons_list_depends, 'list-depends')
+
+
+@click.command()
+@click.option('--database')
+@click.option('--odoo-bin', 'odoo_bin', default=None)
+@click.option('--odoo-config', 'odoo_config',
+              type=click.Path(dir_okay=False, exists=True))
+@click.option('--git-commit', 'git_commit', is_flag=True, default=False)
+@click.option('--languages', default='')
+@click.option('--git-push', 'git_push', is_flag=True, default=False)
+@click.option('--languages', default='')
+@click.option('--git-push-branch', 'git_push_branch')
+@click.option('--git-remote-url', 'git_remote_url')
```

Comment:
s/git-remote-url/git-push-remote-url/ for consistency?

Reply:
or `git-push-remote` tout court: it can be a remote name or a remote url, git push will behave the same (and it can default to origin)

Reply:
Should we change the ```cmd_push``` method defined in ```tools.py``` then?

Reply:
I guess so yes